app/javascript/components/UI/ShareContent.js

```diff
@@ -10,7 +10,7 @@ import {NavLink} from "react-router-dom";
 export const Logo = () => <img src={logo} alt="logo" style={{width: 190, height: 87}} />
 
 export const BigBtnEmotion = ({ emotion, onClick, showPencil = true, addClass = '' }) =>
-  <button className={`${addClass} btn emotion ${emotion.category}`}>
+  <button className={`${addClass} btn-custom emotion ${emotion.category}`}>
```

Comment:
If my upper proposition  approvement to  you then change that on it

`className={${addClass} btn btn-bubbles wb2 ${emotion.category} `

Reply:
@VitaliiShevchenko see previous comment